docs/apache-airflow-providers-apache-druid/connections/index.rst

```diff
@@ -0,0 +1,31 @@
+ .. Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+Connection Types
+================
+
+Apache Airflow implemented connection to Apache Druid in different ways that listed here:
+    * Connection that uses the druid ingestion type that makes it possible to submit index/ingestion jobs.
+      To use the ingestion job, choose ``Druid Ingest Connection``
+    * Connection that uses to Query data using SQL.
+      To Query data use ``Druid Broker Connection``
+
```

Comment:
Looks kind of hard to read in additional to repeatable bullet lists

![image](https://github.com/apache/airflow/assets/3998685/2bf86642-227f-4f17-80b6-918a99fa0711)

If the difference mentioned into the connections, maybe better remove section and keep only toctree? WDYT @eladkal @blcksrx

Reply:
Since the page would be empty, I believe it it good to be for the context and reduce the risk of confusion